### PR TITLE
Telemetry mappings

### PR DESCRIPTION
**Pull request by IotaBread**

### CODE REVIEW

File: mappings/net/minecraft/client/util/telemetry/TelemetryManager.mapping [hunk 1]
```diff
@@ -0,0 +1,44 @@
+CLASS net/minecraft/unmapped/C_wwbyxkgc net/minecraft/client/util/telemetry/TelemetryManager
+	FIELD f_dgegipht session Lcom/mojang/authlib/minecraft/TelemetrySession;
+	FIELD f_dkujvsyq client Lnet/minecraft/unmapped/C_diboefya;
+	FIELD f_egikwkbj TELEMETRY_SENDER_COUNTER Ljava/util/concurrent/atomic/AtomicInteger;
```
Comment (AlsoSylv) [inline mappings/net/minecraft/client/util/telemetry/TelemetryManager.mapping]: What does this do exactly? Feels like there's definitely a better name for it.

File: mappings/net/minecraft/client/util/telemetry/TelemetryManager.mapping [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	FIELD f_ywwymtzy TELEMETRY_SENDER_EXECUTOR Ljava/util/concurrent/Executor;
+	METHOD <init> (Lnet/minecraft/unmapped/C_diboefya;Lcom/mojang/authlib/minecraft/UserApiService;Ljava/util/Optional;Ljava/util/Optional;Ljava/util/UUID;)V
+		ARG 1 client
+		ARG 2 userApiService
+		ARG 3 userId
+		ARG 4 clientId
+		ARG 5 deviceSessionId
+	METHOD m_aiwjwbjb (Ljava/lang/Runnable;)Ljava/lang/Thread;
+		ARG 0 action
+	METHOD m_ajbqkwpi setServerBrandAndSend (Ljava/lang/String;)V
```
Comment (AlsoSylv) [inline mappings/net/minecraft/client/util/telemetry/TelemetryManager.mapping]: Why not just `setServerBrand`?

File: mappings/net/minecraft/client/util/telemetry/TelemetryManager.mapping [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+		ARG 1 serverBrand
+	METHOD m_bshxffst getServerType ()Ljava/lang/String;
+	METHOD m_ejajndgj setGameModeAndSend (Lnet/minecraft/unmapped/C_lvypchgl;Z)V
```
Comment (AlsoSylv) [inline mappings/net/minecraft/client/util/telemetry/TelemetryManager.mapping]: Same here

File: mappings/net/minecraft/client/util/telemetry/TelemetryManager.mapping [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+		ARG 1 client
+		ARG 2 userApiService
+		ARG 3 userId
+		ARG 4 clientId
+		ARG 5 deviceSessionId
+	METHOD m_aiwjwbjb (Ljava/lang/Runnable;)Ljava/lang/Thread;
+		ARG 0 action
+	METHOD m_ajbqkwpi setServerBrandAndSend (Ljava/lang/String;)V
+		ARG 1 serverBrand
+	METHOD m_bshxffst getServerType ()Ljava/lang/String;
```
Comment (AlsoSylv) [inline mappings/net/minecraft/client/util/telemetry/TelemetryManager.mapping:19]: If those two shouldn't change, shouldn't this have `AndSend` at the end?

File: mappings/net/minecraft/client/util/telemetry/TelemetryManager.mapping (hunk 1 above)
Comment (IotaBread) [reply mappings/net/minecraft/client/util/telemetry/TelemetryManager.mapping]: It counts the number of threads in `TELEMETRY_SENDER_EXECUTOR`. Any suggestions for a name?

File: mappings/net/minecraft/client/util/telemetry/TelemetryManager.mapping (hunk 2 above)
Comment (IotaBread) [reply mappings/net/minecraft/client/util/telemetry/TelemetryManager.mapping]: It also executes `TelemetryManager#send(GameModeData)`

File: mappings/net/minecraft/client/util/telemetry/TelemetryManager.mapping (hunk 4 above)
Comment (IotaBread) [reply mappings/net/minecraft/client/util/telemetry/TelemetryManager.mapping:19]: Nope, it just gets the server type as a string, but it doesn't send anything

**Review by NoComment1105 — APPROVED** [review]

File: mappings/net/minecraft/client/util/telemetry/TelemetryManager.mapping (hunk 2 above)
Comment (Jamalam360) [reply mappings/net/minecraft/client/util/telemetry/TelemetryManager.mapping]: I haven't read the conventions yet but would something like `updateServerBrand` fit? A bit less wordy and still descriptive

**Review by Jamalam360 — APPROVED** [review]
Looks good

File: mappings/net/minecraft/client/util/telemetry/TelemetryManager.mapping (hunk 1 above)
Comment (OroArmor) [reply mappings/net/minecraft/client/util/telemetry/TelemetryManager.mapping]: While the field is `static final`, i feel like Atomic values have different usecases than actual `static final` fields. I would consider naming this `telemetrySenderThreads`

**Review by OroArmor — APPROVED** [review]

**Review by AlsoSylv — APPROVED** [review]
Looks good to me

**Pull request merged**